README.md

```diff
@@ -325,21 +325,22 @@ You can also install the extension without access to the internet by following t
 
 ## Configuration
 
-The Puppet VSCode extension can use the PDK or the Puppet Agent to provide advanced features, which can be set using two configuration options: `puppet.installType` and `puppet.installDirectory`.
+### Puppet Source Configuration
 
-- By default the `puppet.installType` is set to `agent`, and allows you to choose to use `pdk` or `agent`.
-- The `puppet.installDirectory` allows you to choose a custom path to either a PDK install or a Puppet-Agent install, and must match the correct setting in `puppet.installType`.
+The Puppet VSCode extension can use either the `PDK` or the `Puppet Agent` as a source to provide advanced features. Which source it uses, and what location to find them in, can be set using two configuration options: `puppet.installType` and `puppet.installDirectory`.
 
-If no settings are chosen, the extension will attempt to use the Puppet Agent, to force it to use the PDK you have to select `pdk` for `puppet.installType`.
+- By default the `puppet.installType` is set to `auto`, and allows you to choose to use `auto`, `pdk` or `agent`.
+- The `puppet.installDirectory` allows you to choose a custom path to either a PDK install or a Puppet-Agent install, and must match the correct setting in `puppet.installType`.
 
+> The default values for these settings will require no configuration by the user in order to use the extension if the PDK or Puppet Agent are installed in the default locations.
 
-> Note: The `puppet.installDirectory` setting name will be changed in the `0.12.0` release to `puppet.installDirectory`
+#### Automatic Configuration
 
-> Note: Only PDK v1.5.0 and above are supported.
+By default the extension attempts to automatically find a valid installation of the `PDK` on your system. If a `PDK` installation is not found, it attempts to find a `Puppet Agent` installation. The locations it looks for are the default install locations for each product. Exact default values for these locations can be found in the following sections for manually configuring the extension.
```

Comment:
nit: Do we really need to use backticks for PDK?  Seems overkill

Reply:
Removed